### PR TITLE
[13.0][IMP] l10n_th_th_withholding_tax_cert_form

### PR DESCRIPTION
**Pull request by komsan-S**
add action send by email

### CODE REVIEW

**Comment by kittiu** [comment]
> add action send by email

I don't know what this is for. @komsan-S can you explain more, or attach some picture so I know what it is.
Note: Thai language is ok to explain, but English is preferred. Thanks.

**Comment by komsan-S** [comment]
@kittiu  Ok thank for commented. I add action button send by email. and I create template for withholding tax cax. You can use it for send email.
Screenshot from 2020-11-24 15-36-58

**Comment by kittiu** [comment]
@Saran440 can you help review/approve this?

File: l10n_th_withholding_tax_cert_form/data/mail_template.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <!--Email template -->
+        <record id="withholding_tax_cert_form_email_template" model="mail.template">
+            <field name="name">Withholding Tax Cert : Send by email</field>
+            <field name="model_id" ref="l10n_th_withholding_tax_cert.model_withholding_tax_cert"/>
+            <field name="subject">${object.supplier_taxid}</field>
+            <field name="partner_to">${object.partner_ids}</field>
+            <field name="body_html" type="xml">
+                <div style="margin: 0px; padding: 0px;">
+                    <p style="margin: 0px; padding: 0px; font-size: 13px;">
+                        Dear ${object.supplier_partner_id.name}.I'm Attach withholdding tax pdf report with email.Please check the validity of the file.<br/>
```
Comment (flook-chaiwat) [inline l10n_th_withholding_tax_cert_form/data/mail_template.xml]:
```suggestion
                        Dear ${object.supplier_partner_id.name}.
                        <br /><br />
                        Here is your <strong>Withholding Tax Certificate</strong>. Please check the validity of the attachment.
                        <br /><br />
                        Do not hesitate to contact us if you have any questions.<br/><br/>
```

Comment (flook-chaiwat) [reply l10n_th_withholding_tax_cert_form/data/mail_template.xml]: I suggest changing the mail-template.

Comment (newtratip) [reply l10n_th_withholding_tax_cert_form/data/mail_template.xml]: Fixed.
@flook-chaiwat , Thanks for your suggestion.

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Pull request merged**